src/adaptation/Adaptation.jl

```diff
@@ -23,6 +23,7 @@ adapt!(
     α::AbstractScalarOrVec{<:AbstractFloat}
 ) where {T<:AbstractAdaptor} = error("`adapt!(adaptor::$T, θ::AbstractVecOrMat{<:AbstractFloat}, α::AbstractScalarOrVec{<:AbstractFloat})` is not implemented.")
 reset!(adaptor::T) where {T<:AbstractAdaptor} = error("`reset!(adaptor::$T)` is not implemented.")
+init!(adaptor::T, n_adapts::Int) where {T<:AbstractAdaptor} = throw(MethodError(init!, adaptor, n_adapts))
```

Comment:
I suggest we use `initialize` instead of `init`, since it is not a long word anyway. Similar for all cases below.

Reply:
Sure.